modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -162,8 +162,16 @@ include::example$EncryptingUsingSDK.java[tag=encrypting_using_sdk_9,indent=0]
 [#migration-from-sdk2]
 == Migrating from SDK 2
 
-If you were previously using Field-Level Encryption with Java SDK 2, a few extra configuration steps are required.
+SDK 3 can be configured to read fields encrypted by SDK 2.
+However...
 
+WARNING: SDK 2 cannot read fields encrypted by SDK 3.
```

Comment:
I think there's a way for a callout box that you could move this above the paragraph, then drop the however…

(also, ellipses are actually a unicode thing!)